itest/litd_custom_channels_test.go

```diff
@@ -1805,7 +1809,7 @@ func testCustomChannelsLiquidityEdgeCases(_ context.Context,
 			SatPerVByte: 5,
 		},
 	)
-	defer closeChannelAndAssert(t, net, dave, channelOp, false)
+	defer closeChannelAndAssert(t, net, dave, channelOp, true)
```

Comment:
Why do we need to force close here now?

Reply:
The very last edge case leaves a payment hanging (i.e waiting for timeout) so the channel would not coop close as there were still in-flight HTLCs. I'll see if there's a way to bypass this and wait for the htlcs to get cancelled